deployment/legacy-migration/maintenance-worker/migration/neo4j/contributions.cql

```diff
@@ -28,7 +28,7 @@
 [?]     unique: true, // Unique value is not enforced in Nitro?
 [-]     index: true
       },
-[ ]   type: {
+[ ]   type: { // db.getCollection('contributions').distinct('type') -> 'DELETED', 'cando', 'post'
```

Comment:
Why is `DELETED` a type :thinking: ?

Reply:
I cannot tell you - its just sure it is there